src/gui/properties/speedplotview.cpp

```diff
@@ -71,62 +75,94 @@ SpeedPlotView::SpeedPlotView(QWidget *parent)
 void SpeedPlotView::setGraphEnable(GraphID id, bool enable)
 {
     m_properties[id].m_enable = enable;
```

Comment:
I understand that this does not apply to the current PR, but somehow we missed it earlier. 
Don't need to use the `m_` prefix for the public fields of classes/structures (even if this structure itself is declared as private). I mean GraphProperties. Please fix it.